### DISCUSSION TITLE
How to use slash commands

### DISCUSSION THREAD

**Discussion by developers192** [mccoderpy/discord.py-message-components#15] [Q&A] [answered]
I'm using your library, discord.py-message-components. It's there any way to use slash commands with the libraray?

**Comment by mccoderpy** [comment, marked as answer]
> I'm using your library, discord.py-message-components. It's there any way to use slash commands with the libraray?

I am (more or less) working on an implementation of this at the moment. However, since I'm doing it all by myself and for a few motivational reasons, it's taking a while. You can use a slightly adapted version of `discord.py-slash-command` v1.2.2 like I'm still doing for my Music-Bot at the moment.
I could send you the files if you want.

**Reply by developers192** [reply]
Thanks for your response! Guess I'll wait until the next update.

**Reply by mccoderpy** [reply]
Ok, (yet another reason to work faster ) 😂

**Reply by developers192** [reply]
Also can you send me the files you mentioned and how to use them? I want to add slash commands so badly

**Reply by mccoderpy** [reply]
Shure, send me a friend request on Discord so i could send you the files 😂 [`mccuber04#2960`](https://discord.com/users/693088765333471284)

**Reply by developers192** [reply]
Yo, I dropped you a DM. [Rem#3852](https://discord.com/users/702396243548176505)
